contracts/Lens/VenusLens.sol

```diff
@@ -584,14 +584,38 @@ contract VenusLens is ExponentialNoError {
     }
 
     /**
-     * @notice Calculate the total XVS tokens pending or accrued by a user account
+     * @dev Struct for Pending Rewards for per market
+     */
+    struct PendingReward {
```

Comment:
Let's move structs to the top of the file so that we have less conflicts when we enable linting

https://docs.soliditylang.org/en/v0.8.17/style-guide.html#order-of-layout

Reply:
Done